app/routes/news.mdx

```diff
@@ -22,6 +22,34 @@ import { Anchor } from '~/components/Anchor'
 
 ## 2023
 
+    <CollectionItem
+      className="maxw-none"
+      variantComponent={
+        <CollectionCalendarDate datetime={'October 27, 2023'} />
+      }
+    >
+      <CollectionHeading headingLevel="h3">
+        <Anchor>New Swift-BAT/GUANO and IceCube Notices Available via GCN Kafka</Anchor>
+      </CollectionHeading>
+      <CollectionDescription>
+        The Swift-BAT/GUANO, IceCube, and GCN teams are pleased to announce the availability of new GCN notice types via the new GCN in JSON format.  JSON format notices can be streamed via [Kafka](/quickstart). [Self-service email subscription](/user/email) will be available soon.  These new notice types are *not* available via GCN Classic.
```

Comment:
```suggestion
        The Swift-BAT/GUANO, IceCube, and GCN teams are pleased to announce the availability of new GCN notice types via the new GCN in JSON format.  JSON format notices can be streamed via [Kafka](/quickstart). These new notice types are *not* available via GCN Classic.
```

We have _no_ idea when we will be able to deliver JSON alerts over email, if ever.

Reply:
I agree that we do not know when, but we do have to be able to do this.   Many users receive notices via email, and will expect to continue to be able to do so.

Reply:
We should not promise features that do not have a certain delivery date.